google/cloud/aiplatform/pipeline_jobs.py

```diff
@@ -348,6 +348,7 @@ def submit(
             service_account=self._gca_resource.service_account,
             project=self.project,
             location=self.location,
+            credentials=self.credentials,
```

Comment:
Should `credentials` also be passed to `gcs_utils.create_gcs_bucket_for_pipeline_artifacts_if_it_does_not_exist` below?

Reply:
I'm not sure which code location are you talking about. Could you link to that code line?

Reply:
Oops, I wasn't looking at the whole file. This is the right place to pass it in 👍